ingestion-beam/src/main/java/com/mozilla/telemetry/util/Normalize.java

```diff
@@ -124,4 +125,27 @@ public static String appName(String name) {
     return OTHER;
   }
 
+  /**
+   * Normalize a 2-letter country code value to uppercase, otherwise return "Other".
+   *
+   * <p>This function is intended to be used to normalize ISO 3166-1 alpha-2 codes, but we do not
+   * check that the code actually corresponds to an officially assigned country so that we don't
+   * need to worry about our list of codes drifting out of date. The space of two-letter codes
+   * (26 * 26 = 676) is sufficiently small that the impact of invalid codes would likely be small.
+   *
+   * <p>See https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+   *
+   * <p>Previous implementation with a fixed list of allowable codes:
+   * https://github.com/mozilla-services/lua_sandbox_extensions/blob/607b3c7b9def600ddbd373a14f67a55e1c69f7a1/moz_telemetry/modules/moz_telemetry/normalize.lua#L233-L268
+   */
+  public static String countryCode(String code) {
```

Comment:
I'm trying to consistently use "country code" rather than "country" to make it more clear what's expected. I'm planning to add a top-level metadata field in payloads called "normalized_country_code" rather than "normalized_country" following this pattern.

Reply:
As long as we're aware of renames like this when doing validation / comparing datasets across infra this looks fine.